### DISCUSSION TITLE
use left control as $mod

### DISCUSSION THREAD

**Discussion by bart9h** [i3/i3#5637] [Q&A] [answered]
This is the setup I always use on my Linux machines:

- Mate desktop environment, replacing the window manager with i3.
- Mate control panel -> Keyboard -> Layouts -> Options -> Caps Lock behavior -> Make Caps Lock an additional Ctrl
- In .xsession, `xcape -e Caps_Lock=Escape`
- In .config/i3/config, `set $mod Mod4`

So I use caps lock as escape when pressed alone, or as control when combined with other keys, and use the "Windows" key as the main modifier for i3 bindings.  It works really well, and I'm used to it.

---------------------------

Yesterday I "inherited" an old IBM Model M keyboard from a coworker that had just retired. (It's for the desktop I use at work, it's not actually mine.) I love this keyboard, and was looking forward to using it. The only problem is that is a 101-keys layout, and lacks the "Windows" key (as wall as the "Menu" key).

As I use Caps Lock as Control, I figured I could use the left control key as the main modifier for i3 bindings. But I can't figure out how to do so. Can you help me?

**Comment by w8ste** [comment]
@bart9h If you want to change your modifier key head to your i3 config: ```cd ~/.config/i3```. Then open your ```config```file with the text-editor of your  choice. One of the first lines should be something like ```set $mod Mod4```. Change it to ```set $mod Control``` like shown in the picture below.
![image](https://github.com/i3/i3/assets/115741870/8871db74-427d-42ad-9857-5f50f9ee5388)

In order for the change to take effect, you need to safe the file an reload. Since you don't have access to your current set modifier, i recommend logging out or rebooting your system. After that, you should be good to go :)

Hope this helps ^^

**Comment by orestisfl** [comment]
Would it be possible to use xmodmap to map that specific control key to Mod4?

**Reply by w8ste** [reply]
@orestisfl According to [this](https://simpleit.rocks/linux/change-modifier-key-in-i3/) guide it is possible, even though i have not tested it out ^^

**Reply by bart9h** [reply, marked as answer]
Yes, it is. Here is the my xmodmap profile for the Model M:

    clear lock
    clear control
    clear mod1
    clear mod4
    keycode 37 = Alt_L
    keycode 64 = Super_L
    keycode 66 = Control_L
    add control = Control_L Control_R
    add mod1 = Alt_R Alt_R Meta_L
    add mod4 = Super_L Super_R

Works beautifully now, but I had to undo all changes in the Mate's control panel.
